### DISCUSSION TITLE
Request for Assistance with Modifying Deployed Environment in "Quickstart" Branch

### DISCUSSION THREAD

**Discussion by dhwani-ankit** [egovernments/DIGIT-DevOps#1817] [Q&A] [answered]
My setup is complete, and now I would like to make some modifications to the code. What is the recommended approach for applying these changes?

I have deployed my environment using the "quickstart" branch of the "deploy-as-code" approach.

**Comment by talele08** [comment, marked as answer]
You can create a docker image of the modified code and deploy the docker image using the helm chart of that service. You can also use the command (from deploy-as-code/deployer folder): go run main.go deploy -e <ENV_NAME> -c <DOCKER_IMAGE_NAME> to deploy your image. It basically does the same thing i.e deploys the image using the service helm chart
